csvcubed/csvcubed/utils/file.py

```diff
@@ -33,3 +35,28 @@ def _ensure_dir_structure_exists(dir_path: Path) -> None:
         _ensure_dir_structure_exists(dir_path.parent)
         _logger.debug("Creating directory %s", dir_path)
         dir_path.mkdir()
+
+
+def read_json_from_file(file_path: Path, log: logging.RootLogger) -> dict:
```

Comment:
Again, I think the log here should be a log created specifically for this module instead of being accepted from the caller.